packages/next-app/package.json

```diff
@@ -22,7 +22,8 @@
     "@radix-ui/colors": "^0.1.8",
     "@radix-ui/react-alert-dialog": "^1.0.3",
     "@radix-ui/react-dialog": "^1.0.3",
-    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-dropdown-menu": "^2.0.4",
```

Comment:
Why do a dropdown menu instead of a select?

Reply:
in that case I think we should aim at having multiple selects, each for each filter, like Google does, and not complicate our life with a very custom component.

Reply:
created an [issue here](https://linear.app/bleu-llc/issue/BAL-178/multiple-selects-instead-of-dropdown)